Client/my-story-illustrator-client/src/header/go_back_button.js

```diff
@@ -0,0 +1,16 @@
+import { IconButton } from '@chakra-ui/react'
+import { BsFillArrowLeftSquareFill } from "react-icons/bs";
+
+function GoBackButton(){
+
+    // TODO test that history.back() works
```

Comment:
history.back is probably not exactly what we want to do here, but this is a good start for now.